setup.py

```diff
@@ -25,5 +25,5 @@
       packages=find_packages(exclude=['tests*']),
       include_package_data=True,
       install_requires=[
-          'wagtail>=2.0',
+          'wagtail>=2.15',
```

Comment:
It would be easier to maintain in the future if we bump this up

Reply:
I can't begin to imagine all the version checks if we supported that far back. Good call